docs/cryoet_data_portal_docsite_quick_start.md

```diff
@@ -17,10 +27,10 @@ python -m venv ./venv
 source ./venv/bin/activate
 ```
 
-Install the latest `cryoet-data-portal` package via pip:
+Install the latest `cryoet_data_portal` package via pip:
 
 ```shell
-pip install -U cryoet-data-portal
+pip install -U cryoet_data_portal
```

Comment:
I believe this should be `pip install cryoet-data-portal`.

Reply:
Updated.